### ISSUE TITLE
How to burn

### ISSUE THREAD

**Issue by gus4rs** [ordinals/ord#3989]
Guess this is more a usability question than a bug.

Trying to burn an inscription with
```
ord wallet burn  --fee-rate 20 --postage 1sat <inscription id> 
```

and am being greeted by 

```
error: JSON-RPC error: RPC error response: RpcError { code: -25, message: "Unspendable output exceeds maximum configured by user (maxburnamount)", data: None }
```

This is a bitcoin core fart preventing sending funds to OP_RETURN. My understanding is this param (```maxburnamount```) should be passed to ```sendrawtransaction```, so nothing I can do.

Is the burn feature useless then?

**Comment by gus4rs** [comment]
If I use  the same burn command above but with ```dry-run``` then copy the psbt, import it in Sparrow and broadcast, it gets confirmed, and apparently it works

**Comment by onchainguy-btc** [comment]
Damn. I think to make this work we would need to add support for `maxburnamount` [here](https://github.com/ordinals/rust-bitcoincore-rpc/blob/958cfbd21f8257f6e17df7d0f7e1675f16338947/client/src/client.rs#L1086).

@raphjaph If you want I can have a look at this together with #3977 this week.

**Comment by raphjaph** [comment]
Ah damn, our `mockcore` tests strike again. @onchainguy-btc yes that would be great. Best to split these different issues into separate PRs for easier review.

**Comment by gus4rs** [comment]
Bro, you seriously didn't try to run the burn command even once for real?  😂

When it comes to Core, an outdated spaghetti full of convoluted flags, work arounds, obnoxious API, this is asking for trouble lol

**Comment by casey** [comment]
lmao gottem

**Comment by raphjaph** [comment]
ded

**Comment by onchainguy-btc** [comment]
💀

**Comment by gus4rs** [comment]
Become a mock disrespectoor

**Comment by raphjaph** [comment]
I'm starting to become one

**Comment by onchainguy-btc** [comment]
Managed to pass the max burn amount on my local ord setup. Now running into this error: 

```
error: JSON-RPC error: RPC error response: RpcError { code: -26, message: "tx-size-small", data: None }
```

Seems to be related to [this issue](https://github.com/bitcoin/bitcoin/issues/18296). Need to investigate further 🤔

**Comment by raphjaph** [comment]
This should now be fixed in #4106
